pkg/kubectl/cmd/cmd.go

```diff
@@ -252,6 +255,12 @@ var (
 
 // NewKubectlCommand creates the `kubectl` command and its nested children.
 func NewKubectlCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cobra.Command {
+	if f := os.Getenv("KUBECTL_FEATURE_GATES"); f != "" {
```

Comment:
how about set feature gates via a subcommand or flag instead of setting env vars?